integration/test_authentication.py

```diff
@@ -20,7 +21,24 @@
 WCS_PORT = "8085"
 
 
+def wait_for_weaviate(url: str):
+    ready_url = url + "/v1/.well-known/ready"
+    for _i in range(0, 30, 5):
+        try:
+            status_code = requests.get(
+                ready_url,
+            ).status_code
+            if status_code != 200:
+                time.sleep(5)
+            else:
+                return
+        except requests.exceptions.ConnectionError:
+            time.sleep(5)
+    raise ConnectionError("Weaviate did not start in time.")
+
+
 def is_auth_enabled(url: str):
+    wait_for_weaviate(url)
```

Comment:
why are you not using the client feature to wait? :)

Reply:
Because is_auth_enabled is called before the client object is initialized.

Reply:
then move the order around?

Reply:
_get_access_token() is used in two of these tests before the client is initialized and it would fail if weaviate is not yet started.

Reply:
I see